### PR TITLE
Fix code generator typo

### PR DESCRIPTION
**Pull request by Joeper214**
## What this patch does to fix the issue.
CodeGenerater -> CodeGenerator

### CODE REVIEW

File: blueoil/converter/generate_project.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 from blueoil.converter.core.config import Config
 from blueoil.converter.core.graph import Graph
 from blueoil.converter.core.params import Params
-from blueoil.converter.code_generater import CodeGenerater
+from blueoil.converter.code_generater import CodeGenerator
```
Comment (primenumber) [inline blueoil/converter/generate_project.py]:
```suggestion
from blueoil.converter.code_generator import CodeGenerator
```

Comment (Joeper214) [reply blueoil/converter/generate_project.py]: @primenumber Thanks!

**Comment by Joeper214** [comment]
@primenumber I updated this PR. Please check again.

**Comment by Joeper214** [comment]
> Readability Approval
> 
> What about `docs/dlk/dlk.rst`?

I have an existing PR related to that. I'll fix it there.

**Comment by tsawada** [comment]
> > Readability Approval
> > What about `docs/dlk/dlk.rst`?
> 
> I have an existing PR related to that. I'll fix it there.

Which PR? I think it should be fixed here.

**Comment by Joeper214** [comment]
> > > Readability Approval
> > > What about `docs/dlk/dlk.rst`?
> > 
> > 
> > I have an existing PR related to that. I'll fix it there.
> 
> Which PR? I think it should be fixed here.

Okay. I'll update it here.

**Pull request merged**